RockPaperScissors/RockPaperScissors.cs

```diff
@@ -6,20 +6,138 @@ class Program
     {
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
-            string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
-            Console.WriteLine(CompareHands(hand1, hand2));
-
-            // leave this command at the end so your program does not close automatically
-            Console.ReadLine();
+            PlayGame();
+            // this command is so your program does not close automatically.
+        }
+
+        public static void PlayGame()
+        {
+            int score1 = 0;
+            int score2 = 0; 
+            bool playing = true; //stays true until you type "exit"
+            int round = 0; //tracks what round you're on.
+            
+
+            Console.WriteLine("Let's Play Rock Paper Scissors!");
+            System.Threading.Thread.Sleep(1000);
+
+            //name the players.
+            Console.WriteLine("What is your name?");
+            string humanName = Console.ReadLine();
+            Console.WriteLine("\nHello, {0}!\n", humanName);
+            System.Threading.Thread.Sleep(1000);
+            Console.WriteLine("What would you like to name your opponent?");
+            string compName = Console.ReadLine();
+
+            Console.WriteLine("Okay, Let's play! Type EXIT at any time to quit. Type carefully as an invalid inmput will result in computer winning!");
+            System.Threading.Thread.Sleep(1000);
+            
+            //Game loop
+            while (playing)
+            {
+                round += 1;
+
+                //get human player's choice.
+                Console.WriteLine("\nRound {0}\nChoose your hand: \"rock\", \"paper\" or \"scissors\":", round);
+                string hand1 = Console.ReadLine().ToLower();
+
+                //see if Exit.
+                if (hand1 == "exit" || hand1 == "quit") {
+                    playing = false;
+                    return;
+                }
+
+                //choose a random number from 1-3 for the computer.
+                Random rndX = new Random();
+                int compChoice = rndX.Next(1,3);
```

Comment:
this will not choose a random number of 1, or 2 since 3 is exclusive.
https://msdn.microsoft.com/en-us/library/2dx6wyd4(v=vs.110).aspx